src/daemon/utils_ssl.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * collectd - src/utils_ssl.c
+ * Copyright (C) 2015  Toni Moreno
+ *
+ * This program is free software; you can redistribute it and/or modify it
```

Comment:
libcurl and [the example code](http://curl.haxx.se/libcurl/c/opensslthreadlock.html) are licensed under the [MIT license](https://collectd.org/wiki/index.php/Category:MIT_License). Would you mind using the same licensing terms here, too?